packages/visual-dev/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import styled from 'styled-components'
+
+import { custom } from './Styles.tsx'
+
+const default_size = {
```

Comment:
Try to avoid leaking styling into the component file. This would be better in `Styles.tsx`

Reply:
Added `default_size` variable into `styles.tsx`.